src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -0,0 +1,9 @@
+{
+    "metrics": [
+        {
+            "name": "vscode_activeregions",
```

Comment:
Why not manage these in the upstream central repo? 

- Then there is a single source of truth,
-  also avoids conflicts (although a `vscode_` prefix/convention also makes conflicts unlikely)
- shared patterns are more likely to be noticed, since they don't need to be pulled up out of the consumer projects. 
    - example: if `activeregions` turns out to be something wanted in most/all projects

Actually, why is `vscode_activeregions` vscode-specific? 

Why do we even need project-specific metric names? Projects that don't need certain metrics can simply no-op them. (And this should be explicit rather than passive)

Reply:
Having multiple active regions is a vscode specific feature so there is no reason for it to leak out into the full list, if we want them in multiple projects they will be pulled into the full list. activeregions is also a metric we added recently to answer a specific question and could be removed in the future.

Reply:
There are some advantages to centralizing everything in any case (see above). And little advantage to fragmenting the source of truth as we are doing here.

> Having multiple active regions is a vscode specific feature so there is no reason for it to leak out into the full list

Then projects that don't need it should opt-out of that metric, see above.

Reply:
I was under the impression that VS Code-specific metrics would still be kept in the common repo, but in a separate file. I'd rather have all metrics there, so that way users have a one-stop shop for seeing what metrics we could potentially capture. Additionally, by moving metrics here, this makes the version number for telemetry in `package.json` less definitive.

Reply:
See PR here: https://github.com/aws/aws-toolkit-common/pull/10 , will keep this file and add to .gitignore for testing purposes